src/useObjectState/index.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+import * as R from 'ramda';
+
+const useObjectState = ({ initialState = {}, useStateDep = useState }) => {
+  const [state, setState] = useStateDep(initialState);
+
+  const update = R.pipe(R.mergeRight(state), setState);
+
+  const updateOne = R.curry(R.pipe(R.objOf, update));
```

Comment:
```suggestion
  const updateOne = R.curry((key, value) => update({ key: value}));
```

I think its a bit too complex use curry, pipe and objOf together